response_operations_ui/__init__.py

```diff
@@ -106,6 +108,8 @@ def create_app(config_name=None):
     login_manager.init_app(app)
     login_manager.login_view = "sign_in_bp.sign_in"
 
+    app.oidc = {}
```

Comment:
New Flask app property (set during initialization) to declare the OIDC implementation to be used.